### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by zakiyousufi**
I have updated my portfolio mobile version (Added the mobile menu) and added these changes 
- Created `index.js` file 
- `index.html` file updated 
- `style.css` file updated 
- Done with project requirement
- No linters error
__________________________________________________________________________________
This project was pair programming but my partner leave the program so I did it solo
hhhhhhh

### CODE REVIEW

**Review by omarsalem7 — COMMENTED** [review]
Hi @zakiyousufi , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🛑 💀 

You should add a screenshot from student success  in pull request for proving  that you partner leave Microverse ,so I can processed with the review. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Kossi-stack — COMMENTED** [review]
Hi @zakiyousufi,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request
You have submitted a project, that has linter errors.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## 

_Invalid Code Review Request **does not count** into the code reviews limit._

File: style.css [hunk 1]
```diff
@@ -35,6 +35,43 @@ nav {
...
+.mobileMenu {
+  position: fixed;
+  top: 0;
+  left: 0;
+  display: none;
+  background-color: #2b2a2a;
+  width: 100%;
+  height: 1000px;
+  padding: 24px;
+}
```
Comment (Lucas-Masaba) [inline style.css:47]: - [ ] Please try to make the design of the pop-up window similar to that in the Figma template. The font size of the text should be larger. The text underlines should be thinner and the close button ought to be smaller 👍 
### Your Implementation
image
### Required 
image

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
# Hello @zakiyousufi  👋 , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Highlights 🍾 
- Linter checks passing ✔️ 
- Pop up window functioning as required 🏄 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 💭 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (ishimwezachee) [reply style.css:47]: - [ OPTIONAL] you've made changes from the previous reviewer, but again I recommend to also change your white color to the one from Figma design.

**Review by ishimwezachee — APPROVED** [review]
Hi @zakiyousufi , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (zakiyousufi) [reply style.css:47]: @[ishimwezachee](https://github.com/ishimwezachee)
color changed

**Pull request merged**